typescript implementation/ascent-service/src/api/routes.ts

```diff
@@ -0,0 +1,140 @@
+            version: properties.version
+        }
+        res.status(200).json(aboutInfo);
+    },
+    method: E_METHOD.GET,
+    endPoint: "/about"
+}
+let createAscent: Route = {
+
+    callback: (req: Request, res: Response) => {
+        if(checkJson(req.body))
+        {
+            sendPostRequest("createAscent", req.body, (responseObject:responseFromDatabase) => {
+
+                res.status(responseObject.code).send(responseObject.message);
+            });
+        }
+        else{
+
+            res.status(400).send("Das Json Object hatte nicht die richtigen Felder");
+        }
+    },
+    method: E_METHOD.POST,
+    endPoint: "/createAscent"
+}
+let getAscentById: Route = {
+
+    callback: (req: Request, res: Response) => {
+        if (req.params.ascentId)
+        {
+            sendGetRequest(`getAscent?search=Id&value=${req.params.ascentId}`, "GET", (responseObject: responseFromDatabase) => {
+                res.status(responseObject.code).send(responseObject.message);
+            });  
+        }
+        else{
+
+            res.status(400).send("Das Json Object hatte nicht die richtigen Felder");
+        } 
+    },
+    method: E_METHOD.GET,
+    endPoint: "/getAscent/:ascentId"
+}
+let deleteAscentById: Route = {
+
+    callback: (req: Request, res: Response) => {
+        if (req.params.ascentId) {
+            sendGetRequest(`deleteAscent?search=Id&value=${req.params.ascentId}`, "DELETE", (responseObject: responseFromDatabase) => {
+                res.status(responseObject.code).send(responseObject.message);
+            });
+        }
+        else {
+            res.status(400).send("Das Json Object hatte nicht die richtigen Felder");
+        } 
+    },
+    method: E_METHOD.DELETE,
+    endPoint: "/deleteAscent/:ascentId"
+}
+let getAscentsBySummitId: Route = {
+
+    callback: async (req: Request, res: Response) => {
+
+        if (req.params.summitId) {
+            
+            sendGetRequest(`getAscent?search=summitId&value=${req.params.summitId}`, "GET", (responseObject: responseFromDatabase) => {
+                res.status(responseObject.code).send(responseObject.message);
+            });  
+        }
+        else {
+
+            res.status(400).send("Das Json Object hatte nicht die richtigen Felder");
+        }  
+    },
+    method: E_METHOD.GET,
+    endPoint: "/getAscentsBySummit/:summitId"
+}
+let getAscentsByUserId: Route = {
+
+    callback: async (req: Request, res: Response) => {
+        
+        if (req.params.userId) {
+            sendGetRequest(`getAscent?search=userId&value=${req.params.userId}`,"GET", (responseObject: responseFromDatabase) => {
+                res.status(responseObject.code).send(responseObject.message);
+            });
+        }
+        else {
+
+            res.status(400).send("Das Json Object hatte nicht die richtigen Felder");
+        } 
+    },
+    method: E_METHOD.GET,
+    endPoint: "/getAscentsByUser/:userId"
+}
+routes.push(getAscentsByUserId);
```

Comment:
Consider putting all the routes into an array/object and then looping over it to push them all into `routes`.